MainModule/Client/Core/Functions.lua

```diff
@@ -1778,6 +1778,7 @@ return function(Vargs, GetEnv)
 			for _, p in props do
 				data[p] = service.UserInputService[p]
 			end
+			data["Resolution"] = workspace.CurrentCamera.ViewportSize.X.." x "..workspace.CurrentCamera.ViewportSize.Y
```

Comment:
This may be a nitpick but I think you should change this to `workspace.CurrentCamera.ViewportSize.X..", "..workspace.CurrentCamera.ViewportSize.Y

Reply:
Most standards interpret display resolutions "X × Y" instead of "X, Y" though.